pkg/v1/providers/tests/unit/windows_test.go

```diff
@@ -50,14 +50,15 @@ var _ = Describe("Windows Ytt Templating", func() {
 			"TKG_DEFAULT_BOM":             "tkg-bom-v1.4.0.yaml",
 			"KUBERNETES_RELEASE":          "v1.21.2---vmware.1-tkg.1",
 			"CORE_DNS_IP":                 "10.64.0.10",
+			"CLUSTER_CIDR":                "192.168.1.0/16",
 		})
 
 		// Printing these values helps for debugging the ytt command if needed
-		fmt.Println(values)
+		// fmt.Println(values)
```

Comment:
Can we remove this if it's no longer needed

Reply:
gone!